package.json

```diff
@@ -49,14 +49,16 @@
   },
   "dependencies": {
     "deepmerge-json": "^1.1.0",
-    "gluegun": "^4.7.1"
+    "gluegun": "^4.7.1",
+    "sharp": "^0.30.4"
```

Comment:
We'll be using sharp for image manipulation. Currently, we use it for sizing and transforming app/launcher icons during the generation process. 

I'll probably be using this a lot more for when we add notification icon generation as well (e.g. android requiring grayscale).